packages/grid/_modules_/grid/hooks/features/density/useGridDensity.tsx

```diff
@@ -13,6 +13,7 @@ import { isDeepEqual } from '../../../utils/utils';
 export const COMPACT_DENSITY_FACTOR = 0.7;
 export const COMFORTABLE_DENSITY_FACTOR = 1.3;
 
+// TODO: revise keeping headerHeight and rowHeight in state for v6
```

Comment:
```suggestion
// TODO v6: revise keeping headerHeight and rowHeight in state
```

Seems to be the most common form for v6 comment in the codebase, will help searching for them.